htmd/tests/run_inline_file_tests.py

```diff
@@ -70,16 +70,23 @@ def containsTests(fname):
     t = time.time()
     print(' ************************  Running "{}"  ************************'.format(f))
     if f.endswith('amber.py') or f.endswith('charmm.py') or f.endswith('preparation.py'):
-        process = run('export PYTHONHASHSEED=1; {} {}'.format(pythonexe, f), shell=True)
+        process = Popen('export PYTHONHASHSEED=1; {} {}'.format(pythonexe, f), shell=True, stdout=PIPE, stderr=STDOUT,
+                        universal_newlines=True)
     else:
-        process = run('{} {}'.format(pythonexe, f), shell=True)
+        process = Popen('{} {}'.format(pythonexe, f), shell=True, stdout=PIPE, stderr=STDOUT,
+                        universal_newlines=True)
+    while True:
+        out = process.stdout.readline()
+        print(out.strip())
+        if len(out) == 0:
```

Comment:
If you are sure this is only 0 when finished ok. What would happen if it just didn't print anything between two iterations?

Reply:
there are some tests that don't print anything, and it breaks the loop in the correct manner. I've ran locally and on Travis, and it prints nicely. But I admit maybe I should have used https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate here.
Since this is a major bug, and this works for now, I'll merge and I'll open an issue on improving this printing.